quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/test/java/org/kie/kogito/quarkus/workflows/GreetRestIT.java

```diff
@@ -67,6 +68,43 @@ void testVersion() {
                 .body("version", is("1.0"));
     }
 
+    @Test
+    void testHtmlResponse() {
+        given()
+                .accept("text/html")
+                .when()
+                .get("/greet")
+                .then()
+                .statusCode(is(200))
+                .contentType(startsWith("text/html"))
+                .body(containsString("SONATAFLOW WORKFLOW ENDPOINT"));
+    }
+
+    @Test
+    void testJsonResponseWithHeader() {
+        given()
+                .accept(ContentType.JSON)
+                .when()
+                .get("/greetdetails")
+                .then()
+                .statusCode(200)
+                .contentType(ContentType.JSON)
+                .body("version", is("1.0"))
+                .body("type", is("SW"));
+    }
+
+    @Test
+    void testJsonResponseWithoutHeader() {
+        given()
+                .when()
+                .get("/greetdetails")
```

Comment:
`greetdetails` workflow is java workflow. We should use `greet`, which is a classic workflow.
```suggestion
                .get("/greet")
```